openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -676,7 +681,7 @@ export const deleteSoftDeletedUser = (username) => {
 
 export const toastNotification = (msg) => {
   cy.get('.Toastify__toast-body').should('be.visible').contains(msg);
-  cy.wait(1000);
+  cy.wait(200);
```

Comment:
Why we are still using hardcoded values?

Reply:
it might make the cypress flaky If the cypress tries to close the success toast as soon as it comes on screen! so we are showing the toast for 200mili sec. and after that, we closing the toast!